config/locales/de.yml

```diff
@@ -620,6 +648,7 @@ de:
     upload_file: Datei hochladen
     time:
       before: "%{time} zuvor"
+    "yes": Ja
```

Comment:
Does this need to be in quotation marks?

Reply:
`yes` and `no` are key words in YAML, they need to be in quotes, because they would otherwise evaluate to `true` and `false` [at runtime](https://makandracards.com/makandra/24809-yaml-keys-like-yes-or-no-evaluate-to-true-and-false).